.github/ISSUE_TEMPLATE/request.yaml

```diff
@@ -13,10 +14,57 @@ body:
       placeholder: Q38076
       description: >-
         If possible get the ID from
-        [Wikidata](https://www.wikidata.org/wiki/Wikidata:Main_Page)
+        [Wikidata](https://www.wikidata.org/wiki/Wikidata:Main_Page).
+        Bonus points: Add the storefinder link as a `Location information url` claim
   - type: textarea
     attributes:
       label: Store finder url(s)
       placeholder: >-
         https://www.mcdonalds.com/gb/en-gb/restaurant-locator.html
         https://www.mcdonalds.com.hk/find-a-restaurant/
+  - type: input
+    attributes:
+      label: Sample store page url
+      description: Add a sample store page url if applicable
+  - type: input
+    attributes:
+      label: Countries?
+      placeholder: AU,CA
+      description: Does this spider cover one or two specific countries? Or is it a worldwide brand?
+  - type: dropdown
+    id: difficulty
+    attributes:
+      label: Difficulty?
+      description: How similar to existing code is this spider? See [API SPIDER](../../docs/API_SPIDER.md) and [HARD WORK SPIDER](../../docs/HARD_WORK_SPIDER.md)
+      options:
+        - Easy - existing storefinders and no customisation
+        - Medium - some customisation or data cleansing
+        - Hard - Hard work spider / Bespoke HTML parsing
+        - Extreme - Obsfucated, brittle behaviour
+  - type: dropdown
+    id: size
+    attributes:
+      label: Number of POI?
+      description: Approximately how many points/locations/branches?
+      options:
+        - 0-10
+        - 10-20
+        - 20-50
```

Comment:
We really shouldn't encourage spiders this small. We can't maintain the current ones. Big brands, important brands.

Reply:
Well, depends on what people consume the map data for. If it's about conserving developer resources then I agree.
If it's about significance...

Is there a favourite pub in my city, or coffee chain started my local people that I would personally go out of my way to map on OSM? Absolutely.

Do these people have the same economy of scale to compete with a multinational? Probably no.
Is going to a local business within, say 5km, with a local supply chain better than a hyper efficient org, like say Shien? I would argue yes in a lot of ways (carbon emissions, local jobs, self sustaining communities, etc)

The maintenance we can automate a lot of - do we need a grim reaper bot for proposing killing off busted spiders? Maybe?

The flip side of this is it's just a feature request. If it's 0-10 stores, do any of us go out of our way to build it? Probably no.
500+; different story.

Reply:
You didn't add an option for 500+ though, you added one for 0-10.

We both spend an enormous amount of time mapping in OSM. I love adding independent shops and small local chains. https://www.openstreetmap.org/changeset/155209416 https://www.openstreetmap.org/changeset/86872194 But OSM and ATP are different. We can't scale to a million spiders, but OSM is already past a millions POIs. I want more independent POIs in ATP, but it can't be via a million spiders.